R/pkg/R/mllib_utils.R

```diff
@@ -80,9 +81,9 @@ predict_internal <- function(object, newData) {
 #' model <- read.ml(path)
 #' }
 #' @note read.ml since 2.0.0
-read.ml <- function(path) {
+read.ml <- function(path, sparkSession = NULL) {
```

Comment:
we should get this from the current default session instead of making it a parameter, like here
https://github.com/apache/spark/blob/master/R/pkg/R/SQLContext.R#L136